entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_service.proto

```diff
@@ -14,6 +14,8 @@ service EntityQueryService {
   }
   rpc update (EntityUpdateRequest) returns (stream ResultSetChunk) {
   }
+  rpc bulkUpdate (BulkEntityUpdateRequest) returns (stream ResultSetChunk) {
```

Comment:
Didnt we decide to add a new service EntityService?

Reply:
We did. But, it's a painful migration, where we have to duplicate lots of implementation code, since the protos won't be common either

I figured we can go ahead and add the current API, and then migrate it as a separate task all together. The migration would still involve lots of duplication